src/App/Property.h

```diff
@@ -67,7 +67,7 @@ class AppExport Property : public Base::Persistence
         Transient = 4, // for property container save
         MaterialEdit = 5, // to turn ON PropertyMaterial edit
         NoMaterialListEdit = 6, // to turn OFF PropertyMaterialList edit
-        Output = 7, // same effect as Prop_Output
+        Output = 8, // same effect as Prop_Output

```

Comment:
Why is Output changed to 8 which is the same as for LockDynamic?
Besides this changing the values of the Status enum is problematic because the OR'ed the value is saved in project files and will result into wrong behaviour when loading an old project file.

Reply:
That's a strange mistake. I didn't intend to change it. Maybe got in somehow by cherry pick. I've done a rebase to fix this and the above build error.